src/hb-face.cc

```diff
@@ -706,9 +707,8 @@ _hb_face_builder_reference_table (hb_face_t *face HB_UNUSED, hb_tag_t tag, void
   if (!tag)
     return _hb_face_builder_data_reference_blob (data);
 
-  hb_face_builder_data_t::table_entry_t *entry = data->tables.lsearch (tag);
-  if (entry)
-    return hb_blob_reference (entry->blob);
+  if (data->tables.has (tag))
+    return hb_blob_reference (data->tables[tag]);
```

Comment:
No need to check. Always return. Hashmap will return `nullptr` if not found, and `blob_reference` is safe on that.